ql/src/java/org/apache/hadoop/hive/ql/optimizer/SharedWorkOptimizer.java

```diff
@@ -17,6 +17,7 @@
  */
 package org.apache.hadoop.hive.ql.optimizer;
 
+import java.io.File;
```

Comment:
yeah...I've some stuff which write out `File`-s for now...and I just remove it at the end of preparing the patch....
it gives easier insights into what happened in the SWO.

I would really like to build it into the system and expose it on the HS2 web interface - it would be very usefull...will get to that as well :)
but first I would like to finish this set of patches :)